pkg/proxy/iptables/proxier.go

```diff
@@ -1604,6 +1604,11 @@ func (proxier *Proxier) syncProxyRules() {
 	proxier.iptablesData.Write(proxier.natChains.Bytes())
 	proxier.iptablesData.Write(proxier.natRules.Bytes())
 
+	numberFilterIptablesRules := utilproxy.CountBytesLines(proxier.filterRules.Bytes())
```

Comment:
really minor point: this still includes COMMIT.  If you moved it up 9 lines, before we add the COMMIT, it would be a tiiiiiiiiny bit more accurate :)

Reply:
O(N+1) XD